docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3.4'
+
+services:
+  mongodb:
```

Comment:
Probably need to see if this can start and finish booting before the rest of the containers start